manual/src/refman/exten.etex

```diff
@@ -123,5 +123,30 @@ that are implemented in OCaml, but not described in chapter \ref{c:refman}.
 %HEVEA\cutname{effects.html}
 \input{effects.tex}
 
+\section{s:memory-model}{Memory model for parallel OCaml programs}
```

Comment:
I've dropped the new section, instead just added a link next to refman link: language extensions section says it documents things not present in refman, and parallelism is neither in refman nor in language extensions.

Reply:
Parallelism and the memory model are in the tutorial section

https://github.com/ocaml/ocaml/blob/75b8d68064472a160aa4ee0eb6029561615c6a80/manual/src/allfiles.etex#L52-L53

![image](https://user-images.githubusercontent.com/410484/207564454-cbb38cd9-19f1-47da-8867-3d19b61263b4.png)

Reply:
I agree with @kayceesrk that parallelism is not in its place in the extension chapter, since it is no more a language construct than threads were in OCaml 4.

Reply:
ok I'll remove it